packages/client/screens/MyProfile/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useInterpret, useSelector } from '@xstate/react';
+import { Button, Text } from 'dripsy';
+import React, { useMemo } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import {
+    AppScreen,
+    AppScreenContainer,
+    AppScreenHeader,
+    Typo,
+} from '../../components/kit';
+import { UserProfileScreenProps } from '../../types';
+import { getFakeUserID } from '../../contexts/SocketContext';
+import { createMyProfileInformationMachine } from '../../machines/myProfileInformationMachine';
+import { MusicPlaylistEditorUsersSearchNavigator } from '../../navigation';
+
+const MyProfileScreen: React.FC<UserProfileScreenProps> = ({ navigation }) => {
+    const insets = useSafeAreaInsets();
+    const userID = getFakeUserID();
+
+    const userProfileInformationMachineConfigured = useMemo(
+        () => createMyProfileInformationMachine(),
+        [],
+    );
+
+    const userProfileInformationService = useInterpret(
+        () => userProfileInformationMachineConfigured,
+    );
```

Comment:
Using `useMemo` to compute a _state machine function creator_ once does the same thing as providing a function to `useInterpret`.

In my previous comments, I meant more something like that:

```suggestion
    const userProfileInformationService = useInterpret(
        () => createMyProfileInformationMachine(),
    );
```

Reply:
Ok I see, only because `createMyProfileInformationMachine` does not take args right ?